packages/consumption/src/modules/requests/incoming/IncomingRequestsController.ts

```diff
@@ -423,7 +423,9 @@ export class IncomingRequestsController extends ConsumptionBaseController {
         const relationship = await this.relationshipResolver.getRelationshipToIdentity(peer);
         // no relationship is for a template request on new relationship
         if (relationship && relationship.status !== RelationshipStatus.Active) {
-            return ValidationResult.error(CoreErrors.requests.noMatchingRelationship(peer.toString()));
+            return ValidationResult.error(
+                CoreErrors.requests.wrongRelationshipStatus(`You cannot decide a request from ${peer.toString()} since the relationship is in status ${relationship.status}.`)
```

Comment:
```suggestion
                CoreErrors.requests.wrongRelationshipStatus(`You cannot decide a request from ${peer.toString()} since the relationship is in status '${relationship.status}'.`)
```

Reply:
Maybe also put the address in 's

Reply:
`since the relationship` 

-> I'd say `since you have a relationship with the peer in status ... but you need an Active relationship.`

Reply:
Also the error has `You cannot decide a request` but the method name does not tell anything about that this method can only be called for decide. Maybe inline this.

Reply:
I guess the ' for consistency, although I don't like them - it's clear without them. And going to active is always the only thing an end-user can do if the status is wrong. I even feel that depending on the UI this may be more confusing if the users only see Accept/Reactivate instead of "Switch to active" as options.